client/src/app/project-registration/project-registration-form/project-registration-form.component.ts

```diff
@@ -49,12 +50,21 @@ export class ProjectRegistrationFormComponent implements OnInit {
               private ingestService: IngestService,
               private alertService: AlertService,
               private loaderService: LoaderService,
-              private schemaService: SchemaService) {
+              private schemaService: SchemaService,
+              private autofillProjectService: AutofillProjectService,
+  ) {
   }
 
   ngOnInit() {
-    this.projectIngestSchema['properties']['content'] = this.projectMetadataSchema;
 
+    const queryParam = this.route.snapshot.queryParamMap;
+
+    if (queryParam.has(Identifier.DOI)) {
+      this.loadProject = false;
```

Comment:
Instead of having a variable flag  `loadProject`, you could use the observable obj and async pipe combination in the template.

Reply:
I saw the change! thanks for applying the changes! it looks good!